unbxdAutosuggest.js

```diff
@@ -1494,6 +1577,9 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 					if (sidelen != 0) {
 						html = '<ul class="unbxd-as-sidecontent">';
 						this.options.sideTpl.forEach(function (key) {
+							if (self.options.sortByLength && (key == 'topQueries' || key == 'keywordSuggestions')) {
```

Comment:
actually, we can use `key !== "SORTED_SUGGESTIONS"`. Overall bundling code size will reduce.